screenfetch-dev

```diff
@@ -2002,16 +2002,17 @@ detectgtk () {
 					fi
 				fi
 				# $HOME/.gtkrc.mine theme detect only
-				if [ -f "$HOME/.gtkrc.mine" ]; then
-					if grep -q "^include" $HOME/.gtkrc.mine; then
-						gtk2Theme=$(grep '^include.*gtkrc' $HOME/.gtkrc.mine | awk -F "/" '{ print $5 }')
+				if [[ -f "$HOME/.gtkrc.mine" || -f "$HOME/.gtkrc-2.0.mine" ]]; then
```

Comment:
If this line returns false, the variable `$MINEGTKRC` will not be set and can't be used in line 2014/2015.
You should also use lowercase letters, as uppercase letters are normally used by environment variables (`$CFLAGS`, `$HOME`, `$LD_LIBRARY_PATH`, ...).